fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -500,6 +500,41 @@ public void testGetRDFSource() throws IOException {
     }
 
     @Test
+    public void testGetRDFSourceWithPreferMinimal() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+
+        final HttpGet getMethod = new HttpGet(serverAddress + id);
+        final String preferHeader = "return=minimal";
+        getMethod.addHeader("Prefer", preferHeader);
+
+        try (final CloseableHttpResponse response = execute(getMethod)) {
+            assertEquals(OK.getStatusCode(), getStatus(response));
+            final Collection<String> preferenceApplied = getHeader(response, "Preference-Applied");
+            assertTrue("Preference-Applied header doesn't matched", preferenceApplied.contains(preferHeader));
+        }
+    }
+
+    @Test
+    public void testGetRDFSourceWithPreferRepresentation() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+
+        final HttpGet getMethod = new HttpGet(serverAddress + id);
+        final String preferHeader = "return=representation;"
+                + " include=\"http://fedora.info/definitions/v4/repository#InboundReferences\";"
+                + " omit=\"http://www.w3.org/ns/ldp#PreferMembership http://www.w3.org/ns/ldp#PreferContainment\"";
+        getMethod.addHeader("Prefer", preferHeader);
+
+        try (final CloseableHttpResponse response = execute(getMethod)) {
+            assertEquals(OK.getStatusCode(), getStatus(response));
+            final Collection<String> preferenceApplied = getHeader(response, "Preference-Applied");
+            System.out.println("Preference-Applied header: " + preferenceApplied.toArray()[0]);
```

Comment:
Please remove the println from the test

Reply:
Thanks @birkland. The println statement is removed, and I also rebased it with master.